hash/dc.xml

```diff
@@ -577,13 +578,14 @@ P830-****-** : Appears to be a Sega part number for promo discs.
 	-->
 
 	<software name="18wheelr">
-		<!-- http://redump.org/disc/18129/
+		<!-- http://redump.org/disc/18129/ -->
+		<!--
 		<rom name="18 Wheeler - American Pro Trucker (USA).gdi" size="225" crc="3460f287" md5="95f6a45db8722322c1a435e76b7de72b" sha1="2d542bef13f37103723c316f99d93d2f7c1daf65"/>
 		<rom name="18 Wheeler - American Pro Trucker (USA) (Track 1).bin" size="1058400" crc="d1a18678" md5="4bd9dce3c5606b086b64b88ef9610eb7" sha1="c7794a4af576c2adfa9f099984ca91839dfbe6c7"/>
 		<rom name="18 Wheeler - American Pro Trucker (USA) (Track 2).bin" size="1589952" crc="8557fcaa" md5="824ff123fbce45b5883962eb3fa43cab" sha1="a3705cb0ce53c7adf906c5af76887406984de6a9"/>
 		<rom name="18 Wheeler - American Pro Trucker (USA) (Track 3).bin" size="1185760800" crc="41d83a31" md5="16b0672b221ed0d0eecb6a9d6e8783f9" sha1="fc4ce908933d90cf3624737349208caca5533700"/>
 		-->
-		<description>18 Wheeler: American Pro Trucker (USA)</description>
+		<description>18 Wheeler - American Pro Trucker (USA)</description>
```

Comment:
Boxart is:
```
      18 Wheeler
American Pro Trucker
```
There's no real dash drawn, so we use `:` as a convention for newline breaks. Revert.